samcli/local/docker/lambda_build_container.py

```diff
@@ -237,5 +238,8 @@ def _convert_to_container_dirs(host_paths_to_convert, host_to_container_path_map
     @staticmethod
     def _get_image(runtime):
         return LambdaBuildContainer._IMAGE_NAME_FORMAT.format(
```

Comment:
This would be easier to understand/read if we combined what we could into one string instead of splitting them. Especially since `_IMAGE_OWNER`, `_IMAGE_NAME_PREFIX`, and `_IMAGE_REGISTRY` are only even used to construct the image uri. 

To make this more readable, I would suggest:
```
__IMAGE_URI_PREFIX = "public.ecr.aws/sam/build-"
return f"{_IMAGE_URI_PREFIX}-{runtime}"
```

Reply:
Updated.